### ISSUE TITLE
Question about Localization Loss

### ISSUE THREAD

**Issue by time06-30am** [Say2L/FSHNet#15]
Dear Author,

I would like to ask why in the Argoverse configuration, the value of 'loc_weight' in self.model_cfg.LOSS_CONFIG.LOSS_WEIGHTS['loc_weight'] is set to 0.25, while in Waymo it is set to 3.0. I noticed that both configurations calculate the loss using loc_loss = loc_loss * self.model_cfg.LOSS_CONFIG.LOSS_WEIGHTS['loc_weight'], without using the 'code_weights' parameter. When I use the Argoverse configuration, the trained data seems to have a large angular error in the bounding boxes. Could you provide some insight on this?
